WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -124,15 +124,34 @@ final class OrdersRootViewController: UIViewController {
     @objc private func displaySearchOrders() {
         analytics.track(.ordersListSearchTapped)
 
-        let searchViewController = SearchViewController<OrderTableViewCell, OrderSearchUICommand>(storeID: siteID,
-                                                                                                  command: OrderSearchUICommand(siteID: siteID),
-                                                                                                  cellType: OrderTableViewCell.self,
-                                                                                                  cellSeparator: .singleLine)
+        let searchViewController = SearchViewController<OrderTableViewCell, OrderSearchUICommand>(
+            storeID: siteID,
+            command: OrderSearchUICommand(siteID: siteID,
+                                          onSelectSearchResult: { [weak self] order, viewController in
+                                              guard let self else { return }
+                                              guard featureFlagService.isFeatureFlagEnabled(.sideBySideViewForOrderForm) else {
+                                                  return presentOrder(order, from: viewController)
+                                              }
+                                              navigateToOrderDetail(order)
+                                              viewController.dismiss(animated: true)
+                                          }),
+            cellType: OrderTableViewCell.self,
+            cellSeparator: .singleLine)
         let navigationController = WooNavigationController(rootViewController: searchViewController)
 
         present(navigationController, animated: true, completion: nil)
     }
 
+    private func presentOrder(_ order: Order, from viewController: UIViewController) {
+        let viewModel = OrderDetailsViewModel(order: order)
+        let detailsViewController = OrderDetailsViewController(viewModel: viewModel)
+
+        viewController.navigationController?.pushViewController(detailsViewController, animated: true)
+        ServiceLocator.analytics.track(event: WooAnalyticsEvent.Orders.orderOpen(
```

Comment:
We can remove the ServiceLocator call, since analytics is already declared in the top level:

```suggestion
        analytics.track(event: WooAnalyticsEvent.Orders.orderOpen(
```